main.go

```diff
@@ -1,21 +1,43 @@
 package main
 
 import (
+	"os"
+	"time"
+
 	"github.com/RedHatInsights/entitlements-api-go/config"
 	"github.com/RedHatInsights/entitlements-api-go/controllers"
 	"github.com/RedHatInsights/entitlements-api-go/logger"
 	"github.com/RedHatInsights/entitlements-api-go/server"
 
 	"github.com/sirupsen/logrus"
+
+	"github.com/getsentry/sentry-go"
 )
 
 func main() {
+
 	// Init the logger first thing
 	logger.InitLogger()
+
+	var dsn string = os.Getenv("SENTRY_DSN")
+
+	if dsn != "" {
+		err := sentry.Init(sentry.ClientOptions{})
+		if err != nil {
+			logger.Log.WithFields(logrus.Fields{"error": err}).Error("Error loading Sentry")
+		}
+	} else {
+	  logger.Log.Info("SENTRY_DSN was not set, skipping Sentry initialization.")
+	}
```

Comment:
What are your thoughts on this? https://github.com/RedHatInsights/entitlements-api-go/pull/59/#discussion_r620293109

Reply:
I think it is useful to register and inform us that Sentry is not working on the instance